test_app/views.py

```diff
@@ -138,9 +140,14 @@ def api_root(request, format=None):
             list_endpoints[url.name.removesuffix('-list')] = reverse(url.name, request=request, format=format)
 
     from ansible_base.api_documentation.urls import api_version_urls as docs_urls
-
-    for url in docs_urls:
-        list_endpoints[url.name] = reverse(url.name, request=request, format=format)
+    from ansible_base.authentication.urls import api_version_urls as authentication_urls
+
+    for url in docs_urls + authentication_urls[1:]:
+        if isinstance(url, URLPattern):
+            try:
+                list_endpoints[url.name] = reverse(url.name, request=request, format=format)
+            except NoReverseMatch:
+                pass
```

Comment:
We have a sorted API view page in gateway, I wonder if we should put that in DAB for others to use? Not relavent to this PR though.

Reply:
This is rather hard to maintain. @fosterseth added it a while ago... and yeah it was needed, but it's a challenge. If there is a code pattern that does this more generally (for top-level links) then I'd love to have that in the main DAB code base for general use. Maybe some sub-selection by app would be nice too.